.github/actions/spelling/expect.txt

```diff
@@ -6,6 +6,7 @@ agg
 aicli
 AICLIC
 ajor
+ALLOC
```

Comment:
We should probably put these in allow.txt

Reply:
The only difference I know of is expect == warning and allow == no warning.  But I have never seen these warnings.

Reply:
You'll see them if an item is no longer used and someone adds an additional misspelling.

Since these are terms, it's reasonable to move them to `allow`.

`ajor` for instance is a good example of something that should be in `expect`. It's there because you're currently using `[M]ajor`, but `ajor` isn't really a word, so if you rewrite `[M]ajor` some other way (or mask it w/ a pattern), you wouldn't want someone else to accidentally write `ajor` (as a typo by dropping the `m`).

You could for instance change:
https://github.com/microsoft/winget-cli/blob/741955e48ce2f069ab323f71e40dd1326654e7ab/src/AppInstallerRepositoryCore/Microsoft/ARPHelper.h#L26
to:
```
        // REG_DWORD (ex. 0xMMmmbbbb, MM: major, mm: minor, bbbb: build)
```
At which point the bot would tell you it noticed `ajor` is no longer present (when it notices someone adding a new unrecognized word, which happens often enough).